nipype/workflows/smri/freesurfer/autorecon3.py

```diff
@@ -0,0 +1,901 @@
+        hemiwf2.connect([(hemi_inputspec2, cortical_parcellation_3, [('smoothwm', 'smoothwm'),
+                                                                     ('aseg_presurf', 'aseg'),
+                                                                     ('cortex_label', 'label'),
+                                                                     ('sphere_reg', 'canonsurf'),
+                                                                     ('curv', 'curv'),
+                                                                     ('sulc', 'sulc'),
+                                                                     ('classifier3', 'classifier')])])
+
+        # Parcellation Statistics 3
+        parcellation_stats_white_3 = parcellation_stats_white.clone(
+            name="Parcellation_Statistics_{0}_3".format(hemisphere))
+        parcellation_stats_white_3.inputs.out_color = 'aparc.annot.DKTatlas40.ctab'
+        parcellation_stats_white_3.inputs.out_table = '{0}.aparc.DKTatlas40.stats'.format(hemisphere)
+        parcellation_stats_white_3.inputs.hemisphere = hemisphere
+
+        hemiwf2.connect([(hemi_inputspec2, parcellation_stats_white_3, [('wm', 'wm'),
+                                                                        ('lh_white', 'lh_white'),
+                                                                        ('rh_white', 'rh_white'),
+                                                                        ('transform', 'transform'),
+                                                                        ('brainmask', 'brainmask'),
+                                                                        ('aseg_presurf', 'aseg'),
+                                                                        ('cortex_label', 'in_cortex'),
+                                                                        ('lh_pial', 'lh_pial'),
+                                                                        ('rh_pial', 'rh_pial'),
+                                                                        ('thickness', 'thickness'),
+                                                                        ('ribbon', 'ribbon'),
+                                                                        ]),
+                         (cortical_parcellation_3, parcellation_stats_white_3, [('out_file', 'in_annotation')])
+                         ])
+
+
+        # WM/GM Contrast
+        contrast = pe.Node(Contrast(), name="WM_GM_Contrast_{0}".format(hemisphere))
+        contrast.inputs.hemisphere = hemisphere
+        contrast.inputs.copy_inputs = True
+
+        hemiwf2.connect([(hemi_inputspec2, contrast, [('orig_mgz', 'orig'),
+                                                      ('rawavg', 'rawavg'),
+                                                      ('{0}_white'.format(hemisphere), 'white'),
+                                                      ('cortex_label'.format(hemisphere), 'cortex'),
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Too many arguments for format string](https://www.codacy.com/app/krzysztof-gorgolewski/nipy_nipype/file/2302210762/issues/source?bid=2767207&fileBranchId=3084409#l470)